.travis.yml

```diff
@@ -7,6 +7,8 @@ matrix:
   include:
     - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT_PPA=yes DOCTEST=yes COLUMNS=80
     - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT_PPA=yes DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT_DEV=yes DOCTEST=no  COLUMNS=80
+    - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT_DEV=yes DOCTEST=no  COLUMNS=80
```

Comment:
Probably bikeshedding but would `OCT=ppa`, `OCT=devsnap`, `OCT=` be more readable instead of `OCT_PPA` and `OCT_DEV`?